simulator/simulator.h

```diff
@@ -18,13 +18,30 @@ namespace et_simulator {
 class Simulator
 {
  private:
+  // Context tree
   CCTree * tree;
+  int currentMethod;
```

Comment:
This is not used anywhere, right?

Reply:
Good catch. This was replaced by a variable in cctree. I forgot to delete this.